### PR TITLE
Added code for Squareroot(python)

### PR DESCRIPTION
**Pull request by Sathwika-02**
@tarun26091999 @rudrakshi99 Please review this PR .This is solve issue #920

### CODE REVIEW

**Comment by rish-singhal** [comment]
please resolve the conflicts, also use pylint once and try to get the score to 10

File: Code/Python/squareroot.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            high=middle-1
+
+    return result
+n=int(input("enter a number :"))
+print(squareroot(n))
+
+"""
+Time complexity:O(logn)
+space complexity: O(1)
+
```
Comment (rudrakshi99) [inline Code/Python/squareroot.py:41]: add complexities at the end

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

File: Algorithm/Number_theory/readme.md [hunk 2]
```diff
@@ -32,16 +32,18 @@
...
-
 ## Questions related to Number Theory
 
+- Counting number of digits in factorial using Kamenetsky's Formula ----> [Java](Code/Java/Kamenetsky_Formula.java)
+- GCD of two number using Euclidean Algorithm ----> [Java](/Code/Java/GCD.java)
 - All Unique Prime Factors ----> [Java](/Code/Java/Uniquefactor.java)
 - Check Prime Number ----> [Java](/Code/Java/check_prime.java)
 - Check Trimorphic number ---->[C++](/Code/C++/Trimorphic_number.cpp)
 - Checking for Anagram ---->[Python](/Code/Python/Anagram.py)
 - Counting number of digits in factorial using Kamenetsky's Formula ----> [Java](Code/Java/Kamenetsky_Formula.java)
```
Comment (rish-singhal) [inline Algorithm/Number_theory/readme.md:43]: combine and sort this list!

**Review by rish-singhal — CHANGES_REQUESTED** [review]

File: Algorithm/Number_theory/readme.md [hunk 3]
```diff
@@ -32,47 +32,47 @@
...
 
 - All Unique Prime Factors ----> [Java](/Code/Java/Uniquefactor.java)
 - Check Prime Number ----> [Java](/Code/Java/check_prime.java)
+- Check Triautomorphic number ----> [Java](/Code/Java/triautomorphic_number.java)
 - Check Trimorphic number ---->[C++](/Code/C++/Trimorphic_number.cpp)
 - Checking for Anagram ---->[Python](/Code/Python/Anagram.py)
 - Counting number of digits in factorial using Kamenetsky's Formula ----> [Java](Code/Java/Kamenetsky_Formula.java)
 - Determining if a number is a prime number or not using Trial Division Algorithm ---> [C++](/Code/C++/prime_number.cpp)
 - Factorial of a number without recursion ---->[Python](/Code/Python/factorial_without_recursion.py)
 - Fibonacci Series ----> [Java](/Code/Java/Fibonacci_Series.java)
```
Comment (rish-singhal) [inline Algorithm/Number_theory/readme.md:45]: Why there are 2 headings with the same name. Merge them.

**Review by rish-singhal — CHANGES_REQUESTED** [review]

File: Algorithm/Number_theory/readme.md (hunk 3 above)
Comment (rish-singhal) [reply Algorithm/Number_theory/readme.md:45]: remove the above one!

**Comment by rish-singhal** [comment]
Not a valid PR

**Pull request closed without merging**